libs/sr/dm-localization/src/executors/gen-locale/lib/stringData.ts

```diff
@@ -61,6 +61,32 @@ function processString(str: string | undefined) {
   return str
 }
 
+function createValueStr(
+  index: string | null,
+  type: string | null,
+  suffix: string | null,
+  plainString: string | null
+) {
+  if (index) {
```

Comment:
can `index` be `0` here?

Reply:
Doesn't seem so, and I had to add `+1` to our interpolation object to handle that